docs/reference/components/sources/prometheus_scrape.cue

```diff
@@ -1,6 +1,6 @@
 package metadata
 
-components: sources: prometheus: {
+components: sources: prometheus_scrape: {
```

Comment:
If the `prometheus` name will still be supported? If so, add an `alias` field here. We're not yet sure what we'll do with that information in the website but it'll be good to make note. Here's the relevant CUE field: https://github.com/timberio/vector/blob/master/docs/reference/components.cue#L54

Reply:
Yes, the old name is still supported to prevent this from breaking existing configs. I've added the alias field, which is a good idea, and for the `prometheus_exporter` as well.